apps/prairielearn/src/models/questions.ts

```diff
@@ -21,6 +21,7 @@ const QuestionsPageDataSchema = z.object({
   open_issue_count: z.number().default(0),
   topic: TopicSchema,
   tags: z.array(TagSchema).nullable(),
+  shared_publicly: z.boolean().optional(),
```

Comment:
This is more precise:

```suggestion
  shared_publicly: z.boolean().nullable(),
```

Reply:
I don't want nullable, because I want to be able to just leave it off entirely if I want. (e.g. the query `select_public_questions_for_course` doesn't have this), I don't want to have to do `select shared_publicly as null...` in cases when I don't care about it.